source/includes/platform-payments.md.erb

```diff
@@ -14,16 +14,16 @@ Content-Type: application/json
 Authorization: Basic ...
 ```
 
-```http 
+```http
 HTTP/1.1 200 OK
 Content-Type: application/json
 
 [ {
-  "id" : "id",
-  "accountName" : "accountName",
-  "accountNumber" : "accountNumber",
-  "sortCode" : "sortCode",
-  "status" : "status"
+  "id" : "06ee16ce-def4-4433-80f2-ab35256316e1",
+  "accountName" : "Destination account",
+  "accountNumber" : "00004588",
+  "sortCode" : "203002",
```

Comment:
I've used the FPS addressable acc number and sort code wherever possible to slightly mitigate problem of someone copying non FPS addressable numbers and using them in sandbox.